floris/tools/wind_data.py

```diff
@@ -333,6 +333,67 @@ def plot_wind_rose(
 
         return ax
 
+    def assign_ti_using_wd_ws_function(self, func):
+        """
+        Use the passed in function to new assign values to turbulence_intensities
+
+        Args:
+            func (function): Function which accepts wind_directions as its
+                first argument and wind_speeds as second argument and returns
+                turbulence_intensities
+        """
+        self.ti_table = func(self.wd_grid, self.ws_grid)
+        self._build_gridded_and_flattened_version()
+
+    def assign_ti_using_IEC_method(self, Iref=0.08, offset=3.8):
```

Comment:
Iref=0.08 gives a TI of 9.8% at 8 m/s. That seems a little high compared to what is usually used in FLORIS. Maybe we could change this to 0.07?

Reply:
Done!  Thank you!